contrib/gn_module_validation/backend/gn_module_validation/migrations/9a4b4b6f8fe6_add_fct_auto_validation.py

```diff
@@ -0,0 +1,102 @@
+"""add_fct_auto_validation
+
+Revision ID: 9a4b4b6f8fe6
+Revises: 446e902a14e7
+Create Date: 2023-10-25 17:18:04.438706
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "9a4b4b6f8fe6"
+down_revision = "df93a68242ee"
+branch_labels = None
+depends_on = ("f06cc80cc8ba",)  # gn_commons
+
+schema = "gn_profiles"
+fct_name = "fct_auto_validation"
+
+
+def upgrade():
+    op.execute(
+        f"""
+create or replace function {schema}.{fct_name} (
```

Comment:
Consider adding comments to the SQL function to explain its purpose and how it works. This would make the code easier to understand for other developers. [medium]